CONTRIBUTING.md

```diff
@@ -467,6 +467,14 @@ except (ImportError, IOError) as err:
 This rule doesn't apply to the test files since these files are loaded only when
 running tests and in such a case your module and their external dependencies are installed.
 
+**You also need to test your code without the python lib nor the module installed** 
+to be sure other lines of your code don't break the installation.
+
+##### Recommandation
+A good practice is to only call library from a method or function. 
```

Comment:
A good practice is to only call an external library from inside a method or function.